src/extensions/savetoLaserfiche/SavetoLaserficheCommandSet.ts

```diff
@@ -214,44 +206,29 @@ export default class SendToLfCommandSet extends BaseListViewCommandSet<ISendToLf
         }
       );
       const results = await res.json();
-      const responseEncoded = JSON.stringify(results)
-        .split('_x005f_')
-        .join('_');
-      const responseRemoveOdata = responseEncoded.split('OData_').join('');
-      allFieldValueStore = JSON.parse(responseRemoveOdata);
-      console.log(allFieldValueStore);
-      return allFieldValueStore;
+      this.allFieldValueStore = results;
+      console.log(this.allFieldValueStore);
+      return this.allFieldValueStore;
```

Comment:
good question.. let me double check but they were parsing the object as a string, but it didn't need to be because they weren't actually using that part of the object (and the .json() returns an object so no need to JSON.parse it, which may have been where the error was coming from)